Configure CI/CD

akuzminsky opened the pull request.

akuzminsky committed:
Configure CI/CD

akuzminsky committed:
Add python requirements

akuzminsky committed:
oops - forgot to 'Add python requirements'

akuzminsky committed:
Configure linter

akuzminsky committed:
The '-admin' role can assume the '-state-manager' role

akuzminsky committed:
Add role ih-tf-aws-control-289256138624-github

- GHA worker runs as `ih-tf-aws-control-289256138624-github`
- Then it assumes role `ih-tf-aws-control-289256138624-admin` to manager
AWS resources
- It also can assume role `ih-tf-aws-control-289256138624-state-manager`
to edit the state.

infrahouse8 commented:
# State **`s3://infrahouse-aws-control-289256138624/terraform.tfstate`**
## Affected resources counts

|  Success  |    Add |    Change |    Destroy |
|:---------:|-------:|----------:|-----------:|
|     ✅     |      0 |         0 |          0 |
## Affected resources by action


<details>
<summary>STDOUT</summary>

```[command]/home/runner/work/_temp/3390e578-fb89-484e-82c7-09ce68e15dd7/terraform-bin plan -no-color --out=tf.plan
data.aws_iam_policy_document.ih-tf-github-control-assume: Reading...
data.aws_iam_policy_document.ih-tf-terraform-control-assume: Reading...
data.aws_iam_policy_document.ih-tf-aws-control-assume: Reading...
data.aws_iam_policy_document.ih-tf-aws-control-289256138624-github-assume: Reading...
data.aws_iam_policy_document.ih-tf-aws-control-289256138624-github-permissions: Reading...
data.aws_iam_policy_document.ih-tf-terraform-control-assume: Read complete after 0s [id=1154989936]
data.aws_iam_policy_document.ih-tf-github-control-assume: Read complete after 0s [id=1521129737]
data.aws_iam_policy_document.ih-tf-aws-control-assume: Read complete after 0s [id=1268673852]
data.aws_iam_policy_document.ih-tf-aws-control-289256138624-github-assume: Read complete after 0s [id=2022835186]
data.aws_iam_policy_document.ih-tf-aws-control-289256138624-github-permissions: Read complete after 0s [id=1603774485]
aws_iam_role.ih-tf-github-control: Refreshing state... [id=ih-tf-github-control]
aws_iam_role.ih-tf-aws-control-289256138624-github: Refreshing state... [id=ih-tf-aws-control-289256138624-github]
data.aws_iam_policy.administrator-access: Reading...
aws_dynamodb_table.terraform_locks: Refreshing state... [id=infrahouse-terraform-state-locks]
module.buckets["infrahouse-aws-control-303467602807"].aws_s3_bucket.state-bucket: Refreshing state... [id=infrahouse-aws-control-303467602807]
aws_iam_role.ih-tf-terraform-control: Refreshing state... [id=ih-tf-terraform-control]
module.buckets["infrahouse-github-state"].aws_s3_bucket.state-bucket: Refreshing state... [id=infrahouse-github-state]
module.buckets["infrahouse-aws-control-289256138624"].aws_s3_bucket.state-bucket: Refreshing state... [id=infrahouse-aws-control-289256138624]
module.buckets["infrahouse-aws-control-990466748045"].aws_s3_bucket.state-bucket: Refreshing state... [id=infrahouse-aws-control-990466748045]
aws_iam_openid_connect_provider.github: Refreshing state... [id=arn:aws:iam::289256138624:oidc-provider/token.actions.githubusercontent.com]
aws_iam_role.ih-tf-aws-control: Refreshing state... [id=ih-tf-aws-control]
data.aws_iam_policy_document.ih-tf-aws-control-289256138624-admin-assume: Reading...
data.aws_iam_policy_document.ih-tf-aws-control-289256138624-state-manager-assume: Reading...
data.aws_iam_policy_document.ih-tf-aws-control-289256138624-state-manager-assume: Read complete after 0s [id=2704423537]
data.aws_iam_policy_document.ih-tf-aws-control-289256138624-admin-assume: Read complete after 0s [id=413646908]
aws_iam_role.ih-tf-aws-control-289256138624-state-manager: Refreshing state... [id=ih-tf-aws-control-289256138624-state-manager]
aws_iam_role.ih-tf-aws-control-289256138624-admin: Refreshing state... [id=ih-tf-aws-control-289256138624-admin]
data.aws_iam_policy_document.ih-tf-aws-control-289256138624-state-manager-permissions: Reading...
data.aws_iam_policy_document.ih-tf-aws-control-289256138624-state-manager-permissions: Read complete after 0s [id=3970547186]
aws_iam_policy.ih-tf-aws-control-289256138624-state-manager-permissions: Refreshing state... [id=arn:aws:iam::289256138624:policy/terraform-20230701232847347400000001]
aws_iam_role_policy_attachment.ih-tf-aws-control-289256138624-state-manager: Refreshing state... [id=ih-tf-aws-control-289256138624-state-manager-20230701232848037400000002]
data.aws_iam_policy.administrator-access: Read complete after 1s [id=arn:aws:iam::aws:policy/AdministratorAccess]
aws_iam_role_policy_attachment.ih-tf-aws-control-289256138624-admin: Refreshing state... [id=ih-tf-aws-control-289256138624-admin-20230701234510837900000001]
aws_iam_role_policy_attachment.ih-tf-terraform-control: Refreshing state... [id=ih-tf-terraform-control-20230627172733931200000001]
aws_iam_role_policy_attachment.ih-tf-aws-control: Refreshing state... [id=ih-tf-aws-control-20230630003530530900000001]
aws_iam_role_policy_attachment.ih-tf-github-control: Refreshing state... [id=ih-tf-github-control-20230627172734370400000002]
module.buckets["infrahouse-aws-control-990466748045"].aws_s3_bucket_public_access_block.public_access: Refreshing state... [id=infrahouse-aws-control-990466748045]
module.buckets["infrahouse-aws-control-303467602807"].aws_s3_bucket_versioning.enabled: Refreshing state... [id=infrahouse-aws-control-303467602807]
module.buckets["infrahouse-aws-control-289256138624"].aws_s3_bucket_versioning.enabled: Refreshing state... [id=infrahouse-aws-control-289256138624]
module.buckets["infrahouse-aws-control-303467602807"].aws_s3_bucket_server_side_encryption_configuration.default: Refreshing state... [id=infrahouse-aws-control-303467602807]
module.buckets["infrahouse-github-state"].aws_s3_bucket_versioning.enabled: Refreshing state... [id=infrahouse-github-state]
module.buckets["infrahouse-aws-control-289256138624"].aws_s3_bucket_public_access_block.public_access: Refreshing state... [id=infrahouse-aws-control-289256138624]
module.buckets["infrahouse-github-state"].aws_s3_bucket_public_access_block.public_access: Refreshing state... [id=infrahouse-github-state]
module.buckets["infrahouse-aws-control-289256138624"].aws_s3_bucket_server_side_encryption_configuration.default: Refreshing state... [id=infrahouse-aws-control-289256138624]
module.buckets["infrahouse-aws-control-303467602807"].aws_s3_bucket_public_access_block.public_access: Refreshing state... [id=infrahouse-aws-control-303467602807]
module.buckets["infrahouse-aws-control-990466748045"].aws_s3_bucket_versioning.enabled: Refreshing state... [id=infrahouse-aws-control-990466748045]
module.buckets["infrahouse-aws-control-990466748045"].aws_s3_bucket_server_side_encryption_configuration.default: Refreshing state... [id=infrahouse-aws-control-990466748045]
module.buckets["infrahouse-github-state"].aws_s3_bucket_server_side_encryption_configuration.default: Refreshing state... [id=infrahouse-github-state]

No changes. Your infrastructure matches the configuration.

Terraform has compared your real infrastructure against your configuration
and found no differences, so no changes are needed.
Releasing state lock. This may take a few moments...
::debug::Terraform exited with code 0.
::debug::stdout: data.aws_iam_policy_document.ih-tf-github-control-assume: Reading...%0Adata.aws_iam_policy_document.ih-tf-terraform-control-assume: Reading...%0Adata.aws_iam_policy_document.ih-tf-aws-control-assume: Reading...%0Adata.aws_iam_policy_document.ih-tf-aws-control-289256138624-github-assume: Reading...%0Adata.aws_iam_policy_document.ih-tf-aws-control-289256138624-github-permissions: Reading...%0Adata.aws_iam_policy_document.ih-tf-terraform-control-assume: Read complete after 0s [id=1154989936]%0Adata.aws_iam_policy_document.ih-tf-github-control-assume: Read complete after 0s [id=1521129737]%0Adata.aws_iam_policy_document.ih-tf-aws-control-assume: Read complete after 0s [id=1268673852]%0Adata.aws_iam_policy_document.ih-tf-aws-control-289256138624-github-assume: Read complete after 0s [id=2022835186]%0Adata.aws_iam_policy_document.ih-tf-aws-control-289256138624-github-permissions: Read complete after 0s [id=1603774485]%0Aaws_iam_role.ih-tf-github-control: Refreshing state... [id=ih-tf-github-control]%0Aaws_iam_role.ih-tf-aws-control-289256138624-github: Refreshing state... [id=ih-tf-aws-control-289256138624-github]%0Adata.aws_iam_policy.administrator-access: Reading...%0Aaws_dynamodb_table.terraform_locks: Refreshing state... [id=infrahouse-terraform-state-locks]%0Amodule.buckets["infrahouse-aws-control-303467602807"].aws_s3_bucket.state-bucket: Refreshing state... [id=infrahouse-aws-control-303467602807]%0Aaws_iam_role.ih-tf-terraform-control: Refreshing state... [id=ih-tf-terraform-control]%0Amodule.buckets["infrahouse-github-state"].aws_s3_bucket.state-bucket: Refreshing state... [id=infrahouse-github-state]%0Amodule.buckets["infrahouse-aws-control-289256138624"].aws_s3_bucket.state-bucket: Refreshing state... [id=infrahouse-aws-control-289256138624]%0Amodule.buckets["infrahouse-aws-control-990466748045"].aws_s3_bucket.state-bucket: Refreshing state... [id=infrahouse-aws-control-990466748045]%0Aaws_iam_openid_connect_provider.github: Refreshing state... [id=arn:aws:iam::289256138624:oidc-provider/token.actions.githubusercontent.com]%0Aaws_iam_role.ih-tf-aws-control: Refreshing state... [id=ih-tf-aws-control]%0Adata.aws_iam_policy_document.ih-tf-aws-control-289256138624-admin-assume: Reading...%0Adata.aws_iam_policy_document.ih-tf-aws-control-289256138624-state-manager-assume: Reading...%0Adata.aws_iam_policy_document.ih-tf-aws-control-289256138624-state-manager-assume: Read complete after 0s [id=2704423537]%0Adata.aws_iam_policy_document.ih-tf-aws-control-289256138624-admin-assume: Read complete after 0s [id=413646908]%0Aaws_iam_role.ih-tf-aws-control-289256138624-state-manager: Refreshing state... [id=ih-tf-aws-control-289256138624-state-manager]%0Aaws_iam_role.ih-tf-aws-control-289256138624-admin: Refreshing state... [id=ih-tf-aws-control-289256138624-admin]%0Adata.aws_iam_policy_document.ih-tf-aws-control-289256138624-state-manager-permissions: Reading...%0Adata.aws_iam_policy_document.ih-tf-aws-control-289256138624-state-manager-permissions: Read complete after 0s [id=3970547186]%0Aaws_iam_policy.ih-tf-aws-control-289256138624-state-manager-permissions: Refreshing state... [id=arn:aws:iam::289256138624:policy/terraform-20230701232847347400000001]%0Aaws_iam_role_policy_attachment.ih-tf-aws-control-289256138624-state-manager: Refreshing state... [id=ih-tf-aws-control-289256138624-state-manager-20230701232848037400000002]%0Adata.aws_iam_policy.administrator-access: Read complete after 1s [id=arn:aws:iam::aws:policy/AdministratorAccess]%0Aaws_iam_role_policy_attachment.ih-tf-aws-control-289256138624-admin: Refreshing state... [id=ih-tf-aws-control-289256138624-admin-20230701234510837900000001]%0Aaws_iam_role_policy_attachment.ih-tf-terraform-control: Refreshing state... [id=ih-tf-terraform-control-20230627172733931200000001]%0Aaws_iam_role_policy_attachment.ih-tf-aws-control: Refreshing state... [id=ih-tf-aws-control-20230630003530530900000001]%0Aaws_iam_role_policy_attachment.ih-tf-github-control: Refreshing state... [id=ih-tf-github-control-20230627172734370400000002]%0Amodule.buckets["infrahouse-aws-control-990466748045"].aws_s3_bucket_public_access_block.public_access: Refreshing state... [id=infrahouse-aws-control-990466748045]%0Amodule.buckets["infrahouse-aws-control-303467602807"].aws_s3_bucket_versioning.enabled: Refreshing state... [id=infrahouse-aws-control-303467602807]%0Amodule.buckets["infrahouse-aws-control-289256138624"].aws_s3_bucket_versioning.enabled: Refreshing state... [id=infrahouse-aws-control-289256138624]%0Amodule.buckets["infrahouse-aws-control-303467602807"].aws_s3_bucket_server_side_encryption_configuration.default: Refreshing state... [id=infrahouse-aws-control-303467602807]%0Amodule.buckets["infrahouse-github-state"].aws_s3_bucket_versioning.enabled: Refreshing state... [id=infrahouse-github-state]%0Amodule.buckets["infrahouse-aws-control-289256138624"].aws_s3_bucket_public_access_block.public_access: Refreshing state... [id=infrahouse-aws-control-289256138624]%0Amodule.buckets["infrahouse-github-state"].aws_s3_bucket_public_access_block.public_access: Refreshing state... [id=infrahouse-github-state]%0Amodule.buckets["infrahouse-aws-control-289256138624"].aws_s3_bucket_server_side_encryption_configuration.default: Refreshing state... [id=infrahouse-aws-control-289256138624]%0Amodule.buckets["infrahouse-aws-control-303467602807"].aws_s3_bucket_public_access_block.public_access: Refreshing state... [id=infrahouse-aws-control-303467602807]%0Amodule.buckets["infrahouse-aws-control-990466748045"].aws_s3_bucket_versioning.enabled: Refreshing state... [id=infrahouse-aws-control-990466748045]%0Amodule.buckets["infrahouse-aws-control-990466748045"].aws_s3_bucket_server_side_encryption_configuration.default: Refreshing state... [id=infrahouse-aws-control-990466748045]%0Amodule.buckets["infrahouse-github-state"].aws_s3_bucket_server_side_encryption_configuration.default: Refreshing state... [id=infrahouse-github-state]%0A%0ANo changes. Your infrastructure matches the configuration.%0A%0ATerraform has compared your real infrastructure against your configuration%0Aand found no differences, so no changes are needed.%0AReleasing state lock. This may take a few moments...%0A
::debug::stderr: 
::debug::exitcode: 0
```
</details>
<details>
<summary>STDERR</summary>

```no output```
</details>
<details><summary><i>metadata</i></summary>
<p>
```eyJzMzovL2luZnJhaG91c2UtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0L3RlcnJhZm9ybS50ZnN0YXRlIjogeyJzdWNjZXNzIjogdHJ1ZSwgInN0ZG91dCI6ICJbY29tbWFuZF0vaG9tZS9ydW5uZXIvd29yay9fdGVtcC8zMzkwZTU3OC1mYjg5LTQ4NGUtODJjNy0wOWNlNjhlMTVkZDcvdGVycmFmb3JtLWJpbiBwbGFuIC1uby1jb2xvciAtLW91dD10Zi5wbGFuXG5kYXRhLmF3c19pYW1fcG9saWN5X2RvY3VtZW50LmloLXRmLWdpdGh1Yi1jb250cm9sLWFzc3VtZTogUmVhZGluZy4uLlxuZGF0YS5hd3NfaWFtX3BvbGljeV9kb2N1bWVudC5paC10Zi10ZXJyYWZvcm0tY29udHJvbC1hc3N1bWU6IFJlYWRpbmcuLi5cbmRhdGEuYXdzX2lhbV9wb2xpY3lfZG9jdW1lbnQuaWgtdGYtYXdzLWNvbnRyb2wtYXNzdW1lOiBSZWFkaW5nLi4uXG5kYXRhLmF3c19pYW1fcG9saWN5X2RvY3VtZW50LmloLXRmLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNC1naXRodWItYXNzdW1lOiBSZWFkaW5nLi4uXG5kYXRhLmF3c19pYW1fcG9saWN5X2RvY3VtZW50LmloLXRmLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNC1naXRodWItcGVybWlzc2lvbnM6IFJlYWRpbmcuLi5cbmRhdGEuYXdzX2lhbV9wb2xpY3lfZG9jdW1lbnQuaWgtdGYtdGVycmFmb3JtLWNvbnRyb2wtYXNzdW1lOiBSZWFkIGNvbXBsZXRlIGFmdGVyIDBzIFtpZD0xMTU0OTg5OTM2XVxuZGF0YS5hd3NfaWFtX3BvbGljeV9kb2N1bWVudC5paC10Zi1naXRodWItY29udHJvbC1hc3N1bWU6IFJlYWQgY29tcGxldGUgYWZ0ZXIgMHMgW2lkPTE1MjExMjk3MzddXG5kYXRhLmF3c19pYW1fcG9saWN5X2RvY3VtZW50LmloLXRmLWF3cy1jb250cm9sLWFzc3VtZTogUmVhZCBjb21wbGV0ZSBhZnRlciAwcyBbaWQ9MTI2ODY3Mzg1Ml1cbmRhdGEuYXdzX2lhbV9wb2xpY3lfZG9jdW1lbnQuaWgtdGYtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0LWdpdGh1Yi1hc3N1bWU6IFJlYWQgY29tcGxldGUgYWZ0ZXIgMHMgW2lkPTIwMjI4MzUxODZdXG5kYXRhLmF3c19pYW1fcG9saWN5X2RvY3VtZW50LmloLXRmLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNC1naXRodWItcGVybWlzc2lvbnM6IFJlYWQgY29tcGxldGUgYWZ0ZXIgMHMgW2lkPTE2MDM3NzQ0ODVdXG5hd3NfaWFtX3JvbGUuaWgtdGYtZ2l0aHViLWNvbnRyb2w6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWloLXRmLWdpdGh1Yi1jb250cm9sXVxuYXdzX2lhbV9yb2xlLmloLXRmLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNC1naXRodWI6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWloLXRmLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNC1naXRodWJdXG5kYXRhLmF3c19pYW1fcG9saWN5LmFkbWluaXN0cmF0b3ItYWNjZXNzOiBSZWFkaW5nLi4uXG5hd3NfZHluYW1vZGJfdGFibGUudGVycmFmb3JtX2xvY2tzOiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1pbmZyYWhvdXNlLXRlcnJhZm9ybS1zdGF0ZS1sb2Nrc11cbm1vZHVsZS5idWNrZXRzW1wiaW5mcmFob3VzZS1hd3MtY29udHJvbC0zMDM0Njc2MDI4MDdcIl0uYXdzX3MzX2J1Y2tldC5zdGF0ZS1idWNrZXQ6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWluZnJhaG91c2UtYXdzLWNvbnRyb2wtMzAzNDY3NjAyODA3XVxuYXdzX2lhbV9yb2xlLmloLXRmLXRlcnJhZm9ybS1jb250cm9sOiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1paC10Zi10ZXJyYWZvcm0tY29udHJvbF1cbm1vZHVsZS5idWNrZXRzW1wiaW5mcmFob3VzZS1naXRodWItc3RhdGVcIl0uYXdzX3MzX2J1Y2tldC5zdGF0ZS1idWNrZXQ6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWluZnJhaG91c2UtZ2l0aHViLXN0YXRlXVxubW9kdWxlLmJ1Y2tldHNbXCJpbmZyYWhvdXNlLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNFwiXS5hd3NfczNfYnVja2V0LnN0YXRlLWJ1Y2tldDogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aW5mcmFob3VzZS1hd3MtY29udHJvbC0yODkyNTYxMzg2MjRdXG5tb2R1bGUuYnVja2V0c1tcImluZnJhaG91c2UtYXdzLWNvbnRyb2wtOTkwNDY2NzQ4MDQ1XCJdLmF3c19zM19idWNrZXQuc3RhdGUtYnVja2V0OiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1pbmZyYWhvdXNlLWF3cy1jb250cm9sLTk5MDQ2Njc0ODA0NV1cbmF3c19pYW1fb3BlbmlkX2Nvbm5lY3RfcHJvdmlkZXIuZ2l0aHViOiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1hcm46YXdzOmlhbTo6Mjg5MjU2MTM4NjI0Om9pZGMtcHJvdmlkZXIvdG9rZW4uYWN0aW9ucy5naXRodWJ1c2VyY29udGVudC5jb21dXG5hd3NfaWFtX3JvbGUuaWgtdGYtYXdzLWNvbnRyb2w6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWloLXRmLWF3cy1jb250cm9sXVxuZGF0YS5hd3NfaWFtX3BvbGljeV9kb2N1bWVudC5paC10Zi1hd3MtY29udHJvbC0yODkyNTYxMzg2MjQtYWRtaW4tYXNzdW1lOiBSZWFkaW5nLi4uXG5kYXRhLmF3c19pYW1fcG9saWN5X2RvY3VtZW50LmloLXRmLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNC1zdGF0ZS1tYW5hZ2VyLWFzc3VtZTogUmVhZGluZy4uLlxuZGF0YS5hd3NfaWFtX3BvbGljeV9kb2N1bWVudC5paC10Zi1hd3MtY29udHJvbC0yODkyNTYxMzg2MjQtc3RhdGUtbWFuYWdlci1hc3N1bWU6IFJlYWQgY29tcGxldGUgYWZ0ZXIgMHMgW2lkPTI3MDQ0MjM1MzddXG5kYXRhLmF3c19pYW1fcG9saWN5X2RvY3VtZW50LmloLXRmLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNC1hZG1pbi1hc3N1bWU6IFJlYWQgY29tcGxldGUgYWZ0ZXIgMHMgW2lkPTQxMzY0NjkwOF1cbmF3c19pYW1fcm9sZS5paC10Zi1hd3MtY29udHJvbC0yODkyNTYxMzg2MjQtc3RhdGUtbWFuYWdlcjogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aWgtdGYtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0LXN0YXRlLW1hbmFnZXJdXG5hd3NfaWFtX3JvbGUuaWgtdGYtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0LWFkbWluOiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1paC10Zi1hd3MtY29udHJvbC0yODkyNTYxMzg2MjQtYWRtaW5dXG5kYXRhLmF3c19pYW1fcG9saWN5X2RvY3VtZW50LmloLXRmLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNC1zdGF0ZS1tYW5hZ2VyLXBlcm1pc3Npb25zOiBSZWFkaW5nLi4uXG5kYXRhLmF3c19pYW1fcG9saWN5X2RvY3VtZW50LmloLXRmLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNC1zdGF0ZS1tYW5hZ2VyLXBlcm1pc3Npb25zOiBSZWFkIGNvbXBsZXRlIGFmdGVyIDBzIFtpZD0zOTcwNTQ3MTg2XVxuYXdzX2lhbV9wb2xpY3kuaWgtdGYtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0LXN0YXRlLW1hbmFnZXItcGVybWlzc2lvbnM6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWFybjphd3M6aWFtOjoyODkyNTYxMzg2MjQ6cG9saWN5L3RlcnJhZm9ybS0yMDIzMDcwMTIzMjg0NzM0NzQwMDAwMDAwMV1cbmF3c19pYW1fcm9sZV9wb2xpY3lfYXR0YWNobWVudC5paC10Zi1hd3MtY29udHJvbC0yODkyNTYxMzg2MjQtc3RhdGUtbWFuYWdlcjogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aWgtdGYtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0LXN0YXRlLW1hbmFnZXItMjAyMzA3MDEyMzI4NDgwMzc0MDAwMDAwMDJdXG5kYXRhLmF3c19pYW1fcG9saWN5LmFkbWluaXN0cmF0b3ItYWNjZXNzOiBSZWFkIGNvbXBsZXRlIGFmdGVyIDFzIFtpZD1hcm46YXdzOmlhbTo6YXdzOnBvbGljeS9BZG1pbmlzdHJhdG9yQWNjZXNzXVxuYXdzX2lhbV9yb2xlX3BvbGljeV9hdHRhY2htZW50LmloLXRmLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNC1hZG1pbjogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aWgtdGYtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0LWFkbWluLTIwMjMwNzAxMjM0NTEwODM3OTAwMDAwMDAxXVxuYXdzX2lhbV9yb2xlX3BvbGljeV9hdHRhY2htZW50LmloLXRmLXRlcnJhZm9ybS1jb250cm9sOiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1paC10Zi10ZXJyYWZvcm0tY29udHJvbC0yMDIzMDYyNzE3MjczMzkzMTIwMDAwMDAwMV1cbmF3c19pYW1fcm9sZV9wb2xpY3lfYXR0YWNobWVudC5paC10Zi1hd3MtY29udHJvbDogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aWgtdGYtYXdzLWNvbnRyb2wtMjAyMzA2MzAwMDM1MzA1MzA5MDAwMDAwMDFdXG5hd3NfaWFtX3JvbGVfcG9saWN5X2F0dGFjaG1lbnQuaWgtdGYtZ2l0aHViLWNvbnRyb2w6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWloLXRmLWdpdGh1Yi1jb250cm9sLTIwMjMwNjI3MTcyNzM0MzcwNDAwMDAwMDAyXVxubW9kdWxlLmJ1Y2tldHNbXCJpbmZyYWhvdXNlLWF3cy1jb250cm9sLTk5MDQ2Njc0ODA0NVwiXS5hd3NfczNfYnVja2V0X3B1YmxpY19hY2Nlc3NfYmxvY2sucHVibGljX2FjY2VzczogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aW5mcmFob3VzZS1hd3MtY29udHJvbC05OTA0NjY3NDgwNDVdXG5tb2R1bGUuYnVja2V0c1tcImluZnJhaG91c2UtYXdzLWNvbnRyb2wtMzAzNDY3NjAyODA3XCJdLmF3c19zM19idWNrZXRfdmVyc2lvbmluZy5lbmFibGVkOiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1pbmZyYWhvdXNlLWF3cy1jb250cm9sLTMwMzQ2NzYwMjgwN11cbm1vZHVsZS5idWNrZXRzW1wiaW5mcmFob3VzZS1hd3MtY29udHJvbC0yODkyNTYxMzg2MjRcIl0uYXdzX3MzX2J1Y2tldF92ZXJzaW9uaW5nLmVuYWJsZWQ6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWluZnJhaG91c2UtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0XVxubW9kdWxlLmJ1Y2tldHNbXCJpbmZyYWhvdXNlLWF3cy1jb250cm9sLTMwMzQ2NzYwMjgwN1wiXS5hd3NfczNfYnVja2V0X3NlcnZlcl9zaWRlX2VuY3J5cHRpb25fY29uZmlndXJhdGlvbi5kZWZhdWx0OiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1pbmZyYWhvdXNlLWF3cy1jb250cm9sLTMwMzQ2NzYwMjgwN11cbm1vZHVsZS5idWNrZXRzW1wiaW5mcmFob3VzZS1naXRodWItc3RhdGVcIl0uYXdzX3MzX2J1Y2tldF92ZXJzaW9uaW5nLmVuYWJsZWQ6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWluZnJhaG91c2UtZ2l0aHViLXN0YXRlXVxubW9kdWxlLmJ1Y2tldHNbXCJpbmZyYWhvdXNlLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNFwiXS5hd3NfczNfYnVja2V0X3B1YmxpY19hY2Nlc3NfYmxvY2sucHVibGljX2FjY2VzczogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aW5mcmFob3VzZS1hd3MtY29udHJvbC0yODkyNTYxMzg2MjRdXG5tb2R1bGUuYnVja2V0c1tcImluZnJhaG91c2UtZ2l0aHViLXN0YXRlXCJdLmF3c19zM19idWNrZXRfcHVibGljX2FjY2Vzc19ibG9jay5wdWJsaWNfYWNjZXNzOiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1pbmZyYWhvdXNlLWdpdGh1Yi1zdGF0ZV1cbm1vZHVsZS5idWNrZXRzW1wiaW5mcmFob3VzZS1hd3MtY29udHJvbC0yODkyNTYxMzg2MjRcIl0uYXdzX3MzX2J1Y2tldF9zZXJ2ZXJfc2lkZV9lbmNyeXB0aW9uX2NvbmZpZ3VyYXRpb24uZGVmYXVsdDogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aW5mcmFob3VzZS1hd3MtY29udHJvbC0yODkyNTYxMzg2MjRdXG5tb2R1bGUuYnVja2V0c1tcImluZnJhaG91c2UtYXdzLWNvbnRyb2wtMzAzNDY3NjAyODA3XCJdLmF3c19zM19idWNrZXRfcHVibGljX2FjY2Vzc19ibG9jay5wdWJsaWNfYWNjZXNzOiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1pbmZyYWhvdXNlLWF3cy1jb250cm9sLTMwMzQ2NzYwMjgwN11cbm1vZHVsZS5idWNrZXRzW1wiaW5mcmFob3VzZS1hd3MtY29udHJvbC05OTA0NjY3NDgwNDVcIl0uYXdzX3MzX2J1Y2tldF92ZXJzaW9uaW5nLmVuYWJsZWQ6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWluZnJhaG91c2UtYXdzLWNvbnRyb2wtOTkwNDY2NzQ4MDQ1XVxubW9kdWxlLmJ1Y2tldHNbXCJpbmZyYWhvdXNlLWF3cy1jb250cm9sLTk5MDQ2Njc0ODA0NVwiXS5hd3NfczNfYnVja2V0X3NlcnZlcl9zaWRlX2VuY3J5cHRpb25fY29uZmlndXJhdGlvbi5kZWZhdWx0OiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1pbmZyYWhvdXNlLWF3cy1jb250cm9sLTk5MDQ2Njc0ODA0NV1cbm1vZHVsZS5idWNrZXRzW1wiaW5mcmFob3VzZS1naXRodWItc3RhdGVcIl0uYXdzX3MzX2J1Y2tldF9zZXJ2ZXJfc2lkZV9lbmNyeXB0aW9uX2NvbmZpZ3VyYXRpb24uZGVmYXVsdDogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aW5mcmFob3VzZS1naXRodWItc3RhdGVdXG5cbk5vIGNoYW5nZXMuIFlvdXIgaW5mcmFzdHJ1Y3R1cmUgbWF0Y2hlcyB0aGUgY29uZmlndXJhdGlvbi5cblxuVGVycmFmb3JtIGhhcyBjb21wYXJlZCB5b3VyIHJlYWwgaW5mcmFzdHJ1Y3R1cmUgYWdhaW5zdCB5b3VyIGNvbmZpZ3VyYXRpb25cbmFuZCBmb3VuZCBubyBkaWZmZXJlbmNlcywgc28gbm8gY2hhbmdlcyBhcmUgbmVlZGVkLlxuUmVsZWFzaW5nIHN0YXRlIGxvY2suIFRoaXMgbWF5IHRha2UgYSBmZXcgbW9tZW50cy4uLlxuOjpkZWJ1Zzo6VGVycmFmb3JtIGV4aXRlZCB3aXRoIGNvZGUgMC5cbjo6ZGVidWc6OnN0ZG91dDogZGF0YS5hd3NfaWFtX3BvbGljeV9kb2N1bWVudC5paC10Zi1naXRodWItY29udHJvbC1hc3N1bWU6IFJlYWRpbmcuLi4lMEFkYXRhLmF3c19pYW1fcG9saWN5X2RvY3VtZW50LmloLXRmLXRlcnJhZm9ybS1jb250cm9sLWFzc3VtZTogUmVhZGluZy4uLiUwQWRhdGEuYXdzX2lhbV9wb2xpY3lfZG9jdW1lbnQuaWgtdGYtYXdzLWNvbnRyb2wtYXNzdW1lOiBSZWFkaW5nLi4uJTBBZGF0YS5hd3NfaWFtX3BvbGljeV9kb2N1bWVudC5paC10Zi1hd3MtY29udHJvbC0yODkyNTYxMzg2MjQtZ2l0aHViLWFzc3VtZTogUmVhZGluZy4uLiUwQWRhdGEuYXdzX2lhbV9wb2xpY3lfZG9jdW1lbnQuaWgtdGYtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0LWdpdGh1Yi1wZXJtaXNzaW9uczogUmVhZGluZy4uLiUwQWRhdGEuYXdzX2lhbV9wb2xpY3lfZG9jdW1lbnQuaWgtdGYtdGVycmFmb3JtLWNvbnRyb2wtYXNzdW1lOiBSZWFkIGNvbXBsZXRlIGFmdGVyIDBzIFtpZD0xMTU0OTg5OTM2XSUwQWRhdGEuYXdzX2lhbV9wb2xpY3lfZG9jdW1lbnQuaWgtdGYtZ2l0aHViLWNvbnRyb2wtYXNzdW1lOiBSZWFkIGNvbXBsZXRlIGFmdGVyIDBzIFtpZD0xNTIxMTI5NzM3XSUwQWRhdGEuYXdzX2lhbV9wb2xpY3lfZG9jdW1lbnQuaWgtdGYtYXdzLWNvbnRyb2wtYXNzdW1lOiBSZWFkIGNvbXBsZXRlIGFmdGVyIDBzIFtpZD0xMjY4NjczODUyXSUwQWRhdGEuYXdzX2lhbV9wb2xpY3lfZG9jdW1lbnQuaWgtdGYtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0LWdpdGh1Yi1hc3N1bWU6IFJlYWQgY29tcGxldGUgYWZ0ZXIgMHMgW2lkPTIwMjI4MzUxODZdJTBBZGF0YS5hd3NfaWFtX3BvbGljeV9kb2N1bWVudC5paC10Zi1hd3MtY29udHJvbC0yODkyNTYxMzg2MjQtZ2l0aHViLXBlcm1pc3Npb25zOiBSZWFkIGNvbXBsZXRlIGFmdGVyIDBzIFtpZD0xNjAzNzc0NDg1XSUwQWF3c19pYW1fcm9sZS5paC10Zi1naXRodWItY29udHJvbDogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aWgtdGYtZ2l0aHViLWNvbnRyb2xdJTBBYXdzX2lhbV9yb2xlLmloLXRmLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNC1naXRodWI6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWloLXRmLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNC1naXRodWJdJTBBZGF0YS5hd3NfaWFtX3BvbGljeS5hZG1pbmlzdHJhdG9yLWFjY2VzczogUmVhZGluZy4uLiUwQWF3c19keW5hbW9kYl90YWJsZS50ZXJyYWZvcm1fbG9ja3M6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWluZnJhaG91c2UtdGVycmFmb3JtLXN0YXRlLWxvY2tzXSUwQW1vZHVsZS5idWNrZXRzW1wiaW5mcmFob3VzZS1hd3MtY29udHJvbC0zMDM0Njc2MDI4MDdcIl0uYXdzX3MzX2J1Y2tldC5zdGF0ZS1idWNrZXQ6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWluZnJhaG91c2UtYXdzLWNvbnRyb2wtMzAzNDY3NjAyODA3XSUwQWF3c19pYW1fcm9sZS5paC10Zi10ZXJyYWZvcm0tY29udHJvbDogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aWgtdGYtdGVycmFmb3JtLWNvbnRyb2xdJTBBbW9kdWxlLmJ1Y2tldHNbXCJpbmZyYWhvdXNlLWdpdGh1Yi1zdGF0ZVwiXS5hd3NfczNfYnVja2V0LnN0YXRlLWJ1Y2tldDogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aW5mcmFob3VzZS1naXRodWItc3RhdGVdJTBBbW9kdWxlLmJ1Y2tldHNbXCJpbmZyYWhvdXNlLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNFwiXS5hd3NfczNfYnVja2V0LnN0YXRlLWJ1Y2tldDogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aW5mcmFob3VzZS1hd3MtY29udHJvbC0yODkyNTYxMzg2MjRdJTBBbW9kdWxlLmJ1Y2tldHNbXCJpbmZyYWhvdXNlLWF3cy1jb250cm9sLTk5MDQ2Njc0ODA0NVwiXS5hd3NfczNfYnVja2V0LnN0YXRlLWJ1Y2tldDogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aW5mcmFob3VzZS1hd3MtY29udHJvbC05OTA0NjY3NDgwNDVdJTBBYXdzX2lhbV9vcGVuaWRfY29ubmVjdF9wcm92aWRlci5naXRodWI6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWFybjphd3M6aWFtOjoyODkyNTYxMzg2MjQ6b2lkYy1wcm92aWRlci90b2tlbi5hY3Rpb25zLmdpdGh1YnVzZXJjb250ZW50LmNvbV0lMEFhd3NfaWFtX3JvbGUuaWgtdGYtYXdzLWNvbnRyb2w6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWloLXRmLWF3cy1jb250cm9sXSUwQWRhdGEuYXdzX2lhbV9wb2xpY3lfZG9jdW1lbnQuaWgtdGYtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0LWFkbWluLWFzc3VtZTogUmVhZGluZy4uLiUwQWRhdGEuYXdzX2lhbV9wb2xpY3lfZG9jdW1lbnQuaWgtdGYtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0LXN0YXRlLW1hbmFnZXItYXNzdW1lOiBSZWFkaW5nLi4uJTBBZGF0YS5hd3NfaWFtX3BvbGljeV9kb2N1bWVudC5paC10Zi1hd3MtY29udHJvbC0yODkyNTYxMzg2MjQtc3RhdGUtbWFuYWdlci1hc3N1bWU6IFJlYWQgY29tcGxldGUgYWZ0ZXIgMHMgW2lkPTI3MDQ0MjM1MzddJTBBZGF0YS5hd3NfaWFtX3BvbGljeV9kb2N1bWVudC5paC10Zi1hd3MtY29udHJvbC0yODkyNTYxMzg2MjQtYWRtaW4tYXNzdW1lOiBSZWFkIGNvbXBsZXRlIGFmdGVyIDBzIFtpZD00MTM2NDY5MDhdJTBBYXdzX2lhbV9yb2xlLmloLXRmLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNC1zdGF0ZS1tYW5hZ2VyOiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1paC10Zi1hd3MtY29udHJvbC0yODkyNTYxMzg2MjQtc3RhdGUtbWFuYWdlcl0lMEFhd3NfaWFtX3JvbGUuaWgtdGYtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0LWFkbWluOiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1paC10Zi1hd3MtY29udHJvbC0yODkyNTYxMzg2MjQtYWRtaW5dJTBBZGF0YS5hd3NfaWFtX3BvbGljeV9kb2N1bWVudC5paC10Zi1hd3MtY29udHJvbC0yODkyNTYxMzg2MjQtc3RhdGUtbWFuYWdlci1wZXJtaXNzaW9uczogUmVhZGluZy4uLiUwQWRhdGEuYXdzX2lhbV9wb2xpY3lfZG9jdW1lbnQuaWgtdGYtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0LXN0YXRlLW1hbmFnZXItcGVybWlzc2lvbnM6IFJlYWQgY29tcGxldGUgYWZ0ZXIgMHMgW2lkPTM5NzA1NDcxODZdJTBBYXdzX2lhbV9wb2xpY3kuaWgtdGYtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0LXN0YXRlLW1hbmFnZXItcGVybWlzc2lvbnM6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWFybjphd3M6aWFtOjoyODkyNTYxMzg2MjQ6cG9saWN5L3RlcnJhZm9ybS0yMDIzMDcwMTIzMjg0NzM0NzQwMDAwMDAwMV0lMEFhd3NfaWFtX3JvbGVfcG9saWN5X2F0dGFjaG1lbnQuaWgtdGYtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0LXN0YXRlLW1hbmFnZXI6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWloLXRmLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNC1zdGF0ZS1tYW5hZ2VyLTIwMjMwNzAxMjMyODQ4MDM3NDAwMDAwMDAyXSUwQWRhdGEuYXdzX2lhbV9wb2xpY3kuYWRtaW5pc3RyYXRvci1hY2Nlc3M6IFJlYWQgY29tcGxldGUgYWZ0ZXIgMXMgW2lkPWFybjphd3M6aWFtOjphd3M6cG9saWN5L0FkbWluaXN0cmF0b3JBY2Nlc3NdJTBBYXdzX2lhbV9yb2xlX3BvbGljeV9hdHRhY2htZW50LmloLXRmLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNC1hZG1pbjogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aWgtdGYtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0LWFkbWluLTIwMjMwNzAxMjM0NTEwODM3OTAwMDAwMDAxXSUwQWF3c19pYW1fcm9sZV9wb2xpY3lfYXR0YWNobWVudC5paC10Zi10ZXJyYWZvcm0tY29udHJvbDogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aWgtdGYtdGVycmFmb3JtLWNvbnRyb2wtMjAyMzA2MjcxNzI3MzM5MzEyMDAwMDAwMDFdJTBBYXdzX2lhbV9yb2xlX3BvbGljeV9hdHRhY2htZW50LmloLXRmLWF3cy1jb250cm9sOiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1paC10Zi1hd3MtY29udHJvbC0yMDIzMDYzMDAwMzUzMDUzMDkwMDAwMDAwMV0lMEFhd3NfaWFtX3JvbGVfcG9saWN5X2F0dGFjaG1lbnQuaWgtdGYtZ2l0aHViLWNvbnRyb2w6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWloLXRmLWdpdGh1Yi1jb250cm9sLTIwMjMwNjI3MTcyNzM0MzcwNDAwMDAwMDAyXSUwQW1vZHVsZS5idWNrZXRzW1wiaW5mcmFob3VzZS1hd3MtY29udHJvbC05OTA0NjY3NDgwNDVcIl0uYXdzX3MzX2J1Y2tldF9wdWJsaWNfYWNjZXNzX2Jsb2NrLnB1YmxpY19hY2Nlc3M6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWluZnJhaG91c2UtYXdzLWNvbnRyb2wtOTkwNDY2NzQ4MDQ1XSUwQW1vZHVsZS5idWNrZXRzW1wiaW5mcmFob3VzZS1hd3MtY29udHJvbC0zMDM0Njc2MDI4MDdcIl0uYXdzX3MzX2J1Y2tldF92ZXJzaW9uaW5nLmVuYWJsZWQ6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWluZnJhaG91c2UtYXdzLWNvbnRyb2wtMzAzNDY3NjAyODA3XSUwQW1vZHVsZS5idWNrZXRzW1wiaW5mcmFob3VzZS1hd3MtY29udHJvbC0yODkyNTYxMzg2MjRcIl0uYXdzX3MzX2J1Y2tldF92ZXJzaW9uaW5nLmVuYWJsZWQ6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWluZnJhaG91c2UtYXdzLWNvbnRyb2wtMjg5MjU2MTM4NjI0XSUwQW1vZHVsZS5idWNrZXRzW1wiaW5mcmFob3VzZS1hd3MtY29udHJvbC0zMDM0Njc2MDI4MDdcIl0uYXdzX3MzX2J1Y2tldF9zZXJ2ZXJfc2lkZV9lbmNyeXB0aW9uX2NvbmZpZ3VyYXRpb24uZGVmYXVsdDogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aW5mcmFob3VzZS1hd3MtY29udHJvbC0zMDM0Njc2MDI4MDddJTBBbW9kdWxlLmJ1Y2tldHNbXCJpbmZyYWhvdXNlLWdpdGh1Yi1zdGF0ZVwiXS5hd3NfczNfYnVja2V0X3ZlcnNpb25pbmcuZW5hYmxlZDogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aW5mcmFob3VzZS1naXRodWItc3RhdGVdJTBBbW9kdWxlLmJ1Y2tldHNbXCJpbmZyYWhvdXNlLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNFwiXS5hd3NfczNfYnVja2V0X3B1YmxpY19hY2Nlc3NfYmxvY2sucHVibGljX2FjY2VzczogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aW5mcmFob3VzZS1hd3MtY29udHJvbC0yODkyNTYxMzg2MjRdJTBBbW9kdWxlLmJ1Y2tldHNbXCJpbmZyYWhvdXNlLWdpdGh1Yi1zdGF0ZVwiXS5hd3NfczNfYnVja2V0X3B1YmxpY19hY2Nlc3NfYmxvY2sucHVibGljX2FjY2VzczogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aW5mcmFob3VzZS1naXRodWItc3RhdGVdJTBBbW9kdWxlLmJ1Y2tldHNbXCJpbmZyYWhvdXNlLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNFwiXS5hd3NfczNfYnVja2V0X3NlcnZlcl9zaWRlX2VuY3J5cHRpb25fY29uZmlndXJhdGlvbi5kZWZhdWx0OiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1pbmZyYWhvdXNlLWF3cy1jb250cm9sLTI4OTI1NjEzODYyNF0lMEFtb2R1bGUuYnVja2V0c1tcImluZnJhaG91c2UtYXdzLWNvbnRyb2wtMzAzNDY3NjAyODA3XCJdLmF3c19zM19idWNrZXRfcHVibGljX2FjY2Vzc19ibG9jay5wdWJsaWNfYWNjZXNzOiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1pbmZyYWhvdXNlLWF3cy1jb250cm9sLTMwMzQ2NzYwMjgwN10lMEFtb2R1bGUuYnVja2V0c1tcImluZnJhaG91c2UtYXdzLWNvbnRyb2wtOTkwNDY2NzQ4MDQ1XCJdLmF3c19zM19idWNrZXRfdmVyc2lvbmluZy5lbmFibGVkOiBSZWZyZXNoaW5nIHN0YXRlLi4uIFtpZD1pbmZyYWhvdXNlLWF3cy1jb250cm9sLTk5MDQ2Njc0ODA0NV0lMEFtb2R1bGUuYnVja2V0c1tcImluZnJhaG91c2UtYXdzLWNvbnRyb2wtOTkwNDY2NzQ4MDQ1XCJdLmF3c19zM19idWNrZXRfc2VydmVyX3NpZGVfZW5jcnlwdGlvbl9jb25maWd1cmF0aW9uLmRlZmF1bHQ6IFJlZnJlc2hpbmcgc3RhdGUuLi4gW2lkPWluZnJhaG91c2UtYXdzLWNvbnRyb2wtOTkwNDY2NzQ4MDQ1XSUwQW1vZHVsZS5idWNrZXRzW1wiaW5mcmFob3VzZS1naXRodWItc3RhdGVcIl0uYXdzX3MzX2J1Y2tldF9zZXJ2ZXJfc2lkZV9lbmNyeXB0aW9uX2NvbmZpZ3VyYXRpb24uZGVmYXVsdDogUmVmcmVzaGluZyBzdGF0ZS4uLiBbaWQ9aW5mcmFob3VzZS1naXRodWItc3RhdGVdJTBBJTBBTm8gY2hhbmdlcy4gWW91ciBpbmZyYXN0cnVjdHVyZSBtYXRjaGVzIHRoZSBjb25maWd1cmF0aW9uLiUwQSUwQVRlcnJhZm9ybSBoYXMgY29tcGFyZWQgeW91ciByZWFsIGluZnJhc3RydWN0dXJlIGFnYWluc3QgeW91ciBjb25maWd1cmF0aW9uJTBBYW5kIGZvdW5kIG5vIGRpZmZlcmVuY2VzLCBzbyBubyBjaGFuZ2VzIGFyZSBuZWVkZWQuJTBBUmVsZWFzaW5nIHN0YXRlIGxvY2suIFRoaXMgbWF5IHRha2UgYSBmZXcgbW9tZW50cy4uLiUwQVxuOjpkZWJ1Zzo6c3RkZXJyOiBcbjo6ZGVidWc6OmV4aXRjb2RlOiAwXG4iLCAic3RkZXJyIjogIiIsICJhZGQiOiAwLCAiY2hhbmdlIjogMCwgImRlc3Ryb3kiOiAwfX0=```
</p></details>

akuzminsky committed:
Allow ih-tf-aws-control-289256138624-github edit state bucket

The bucket is `infrahouse-aws-control-289256138624`
The GHA worker needs it to upload a plan.
